LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Radify, Inc.
+All rights reserved.
```

Comment:
Yes. Here is [the template](http://choosealicense.com/licenses/bsd-3-clause/). The first two lines are:
```
Copyright (c) [year], [fullname]
All rights reserved.
```